package.json

```diff
@@ -15,11 +15,11 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.3.1",
-    "@docusaurus/preset-classic": "^2.3.1",
+    "@docusaurus/core": "^2.3.0",
+    "@docusaurus/preset-classic": "^2.3.0",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
-    "docusaurus": "^1.14.7",
+    "docusaurus": "^2.0.0-alpha.378053ac5",
```

Comment:
I think the major part to solve this was to not get any reference to any other docusaurus instance version, as with normal docusaurus installation there is only docusaurus version.

Reply:
> I think the major part to solve this was to not get any reference to any other docusaurus instance version, as with normal docusaurus installation there is only docusaurus version.

I did not get you can you explain me in detail if possible?

Reply:
When you do an upgrade of societies according to the website, the reference to the older version 1 gets inserted.

There is some package we are using that still want V1.

How can we identify what that package is?